src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -186,28 +169,28 @@ public static void TimeoutTest_SendTimeout5Seconds()
     [OuterLoop]
     public static void TimeoutTest_SendTimeout0Seconds()
     {
```

Comment:
Nit: another name that could be improved.  Something like Zero_SendTimeout_Throws_TimeoutException_Immediately.